Cargo.toml

```diff
@@ -23,3 +24,4 @@ url = "2.2.2"
 [dev-dependencies]
 cpal = "0.13.5"
 crossbeam = "0.8.1"
+reqwest = { version = "0.11.10", features = ["stream"] }
```

Comment:
which example requires the `stream` feature? not 100% clear to me

Reply:
It's so that reqwest can read directly from a file. Without the `stream` feature, it would have to be done by reading whole file into memory first.

Reply:
gotcha, that makes sense